XPlat/TensorFlow.Lite/readme.md

```diff
@@ -0,0 +1,138 @@
+# Dotnet new template of the Xamarin Components team
```

Comment:
Can you clean up this file to be more specific to what's actually in the folder? It looks like a template generated file right now.